openmc/__init__.py

```diff
@@ -1,3 +1,5 @@
+from openmc.cell import *
```

Comment:
Are these necessary? I thought they were imported with `openmc.geometry` below, but I may be wrong.